docs/source-2.0/aws/aws-iam.rst

```diff
@@ -715,9 +603,133 @@ explicitly binds ``ActionContextKey1`` to the field ``key``.
     }
 
 
+.. smithy-trait:: aws.iam#disableConditionKeyInference
+.. _aws.iam#disableConditionKeyInference-trait:
+
+``aws.iam#disableConditionKeyInference`` trait
+==============================================
+
+Summary
+    Declares that the condition keys of a resource should not be inferred.
```

Comment:
Q: Is there any documentation of how condition keys are inferred?

Reply:
There's a section titled "Deriving condition keys" below this, referred to as "inferred" throughout the rest of the doc.

Reply:
Could we link that section here? It isnt super clear because of the naming difference.